### DISCUSSION TITLE
Add Configuration for Explore Flip results order

### DISCUSSION THREAD

**Discussion by duhow** [grafana/grafana#42299] [Observability]
I'm suggesting to have a configuration in `grafana.ini`, `[explore]` section, to provide **Flip results order** by default, instead of having to click it for the first query.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
